deploy/role.yaml

```diff
@@ -5,23 +5,25 @@ metadata:
 rules:
 - apiGroups:
   - kubevirt.io
-  - oauth.openshift.io
   - template.openshift.io
   resources:
   - '*'
   verbs:
-  - '*'
+  - create 
+  - get
+  - list
+  - patch
+  - update
+  - watch
+
 - apiGroups:
   - rbac.authorization.k8s.io
   resources:
   - clusterroles
   verbs:
   - create
-  - get
-  - list
   - watch
   - patch
```

Comment:
do we also need "patch"?

Reply:
yep we need it:
`system:serviceaccount:kubevirt:kubevirt-ssp-operator" cannot patch clusterroles.rbac.authorization.k8s.io at the cluster scope: no RBAC policy matched", "reason": "Forbidden"`

Reply:
ok